pkgs/os-specific/darwin/osx-cpu-temp/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub
+, darwin
+}:
+
+stdenv.mkDerivation rec {
+  pname = "osx-cpu-temp";
+  version = "1.0";
+
+  src = fetchFromGitHub rec {
+    name = "${pname}-source";
+    owner = "lavoiesl";
+    repo = pname;
+    rev = "6ec951be449badcb7fb84676bbc2c521e600e844";  # master as of 2020-12-04
+    sha256 = "1nlibgr55bpln6jbdf8vqcp0fj9zv9343vflb7s9w0yh33fsbg9d";
```

Comment:
```suggestion
    owner = "lavoiesl";
    repo = pname;
    rev = "6ec951be449badcb7fb84676bbc2c521e600e844";
    sha256 = "1nlibgr55bpln6jbdf8vqcp0fj9zv9343vflb7s9w0yh33fsbg9d";
```

Reply:
I'd rather keep the changed name for `source`, since this at least gives an visual indication in the nix store for what the entry is, rather than just `/nix/store/<somehash>-source`, which is entirely unhelpful.  Unless you know a reason to avoid this?

Happy to drop the comment though, given the suggested change in version string.